corehq/apps/userreports/tasks.py

```diff
@@ -176,6 +178,58 @@ def _iteratively_build_table(config, resume_helper=None, in_place=False, limit=-
         adapter.after_table_build()
 
 
+@task(queue=UCR_CELERY_QUEUE)
+def compare_ucr_dbs(domain, report_config_id, filter_values, sort_column=None, sort_order=None, params=None):
```

Comment:
those are all used in `_run_report`.

Reply:
Oh. I see now that those aren't passed in because mobile UCR doesn't use those. I didn't add in web reports here, but they would be used for those. 

I'm going to opt to keep those here since it was part of reverting https://github.com/dimagi/commcare-hq/pull/20933/files#diff-1f15a4ea1fec5c4ee5fa155d4d80c3d7L184 and makes it slightly more complete

Reply:
I meant that they aren't passed in to the function anywhere